NewsHSE/Modules/NewsDetail/NewsDetailView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  NewsDetailView.swift
+//  NewsHSE
+//
+//  Created by Darya Zhitova on 20.11.2023.
+//
+
+import UIKit
+import Kingfisher
+
+class NewsDetailView: UIView {
+    
+    private lazy var titleLabel: UILabel = {
```

Comment:
Кстати, lazyness не обязателен. Все равно обращение к вьюхам происходит в инициализаторе(при вызове addSubviews), поэтому сильно в перформансе мы не выигрываем. По сути lazy view есть смысл только в контроллере использовать, т.к. в таком случае мы не нагружаем процесс инициализации контроллера созданием вьюх.
```suggestion
    private let titleLabel: UILabel = {
```

Reply:
пойду заново изучать жц 

со своим swiftui совсем от рук отбилась🥲

Reply:
Только сегодня попытались обсудить холиварную тему переезда АМа на суи.)
Но, кажется, тут дело не в жц, а в том, как работает lazy.

Reply:
не переходить на суи, суи - зло)
так неудобно смотреть комменты через гитхаб, давай шуточки и новости из жизни в телеге обсудим, ахаххаах